frontend/yarn.lock

```diff
@@ -14216,9 +14216,9 @@ react-popper@^1.3.3:
     warning "^4.0.2"
 
 react-range@^1.6.0:
-  version "1.6.0"
-  resolved "https://registry.yarnpkg.com/react-range/-/react-range-1.6.0.tgz#9f7435402fbcf221fb178f4bbdc6a36052c89da0"
-  integrity sha512-OTaylp3wzDxWq1awwH+rOz7UHbKmimu4wmxyZ0XzWtuO19Frz2D8c79R4YTC2IHqX+6ly3YmTLKhICSXi8x/NQ==
+  version "1.6.7"
```

Comment:
This is a dependency in `baseweb` that contains a bug fix to avoid breaking things.